tests/integration/UploadSlideBackTranslationIntegrationTest.php

```diff
@@ -0,0 +1,317 @@
+        self::$filesRoot = $GLOBALS['filesRoot'];
+        $templateDir = self::$filesRoot . "/Templates";
+        self::$fileSystem->mkdir($templateDir);
+        // copy test story template files to files root
+        self::$fileSystem->mirror(dirname(__DIR__) . "/data/templates", $templateDir);
+        self::$uploadedProjectDir = self::$filesRoot . "/Projects";
+
+        self::$db = new PDO(DB_DNS);
+        self::$db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+    }
+
+    public static function tearDownAfterClass(): void
+    {
+        self::$fileSystem->remove(self::$filesRoot);
+        self::$process->stop();
+        self::$db->query('DELETE FROM Slide');
+        self::$db->query('DELETE FROM Stories');
+        self::$db = null;
+    }
+
+
+    public function setUp(): void
+    {
+        $this->httpClient = new Client(['http_errors' => false]);
+    }
+
+    /**
+     *
+     * @return int
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     * @throws \Exception
+     */
+    public function testCreateNewStory(): void
+    {
+        // request payload
+        $payload = [
+            'Key' => '',
+            'PhoneId' => 'rstuvw',
+            'Data' => base64_encode('Test data for story file'),
+            'TemplateTitle' => $this->storyTemplate,
+            'IsWholeStory' => "true"
+        ];
+
+        $response = $this->httpClient->request("POST", self::$host . $this->uri, ['form_params' => $payload]);
+        $responseJson = $response->getBody()->getContents();
+
+        $this->assertEquals(200, $response->getStatusCode());
+        $responseArr = json_decode($responseJson, true);
+        $this->assertArrayHasKey('StoryId', $responseArr);
+        $storyId = $responseArr['StoryId'];
+
+
+        # verify story is created correctly in the database
+        $stories = $this->getStories($storyId);
+        $this->assertCount(1, $stories);
+
+        $this->assertEquals([
+            "id" => $storyId,
+            "title" => "test_story",
+            "language" => "",
+            "projectId" => $this->getProjectId($payload['PhoneId']),
+            "note" => "",
+            'FirstThreshold' => null,
+            'SecondThreshold' => null,
+        ], $stories[0]);
+
+
+        # verify slides are all created correctly in the database
+        $slides = $this->getSlides($storyId);
+        $this->assertCount(3, $slides);
+        $this->assertEquals(0, $slides[0]['slideNumber']);
+        $this->assertEquals(1, $slides[1]['slideNumber']);
+        # In test `story.json`, third slide type is `COPYRIGHT` which is skipped as per implementation
+        $this->assertEquals(3, $slides[2]['slideNumber']);
+
+        $storyFile = sprintf("%s/%s/%s/wholeStory.m4a", self::$uploadedProjectDir, $payload['PhoneId'], $storyId);
+        $this->assertFileExists($storyFile);
+        $this->assertEquals(file_get_contents($storyFile), base64_decode($payload['Data']));
```

Comment:
Can you create a constant for the contents, e.g. `STORY_FILE_CONTENTS`, so that this line is:
`$this->assertEquals(file_get_contents($storyFile), STORY_FILE_CONTENTS);`? 

The payload can be initialed as: `'Data' => base64_encode(STORY_FILE_CONTENTS),`